e2e/vm/config_windows_test.go

```diff
@@ -0,0 +1,12 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+//go:build windows
+
+package vm
+
+import (
+	"os"
+	"path/filepath"
+)
+
+var finchConfigFilePath = filepath.Join(os.Getenv("LOCALAPPDATA")+".finch", "finch.yaml")

```

Comment:
nit: wouldn't this also work?
```suggestion
var finchConfigFilePath = filepath.Join(os.Getenv("LOCALAPPDATA"), ".finch", "finch.yaml")

```

Reply:
Updated.